packages/cli/generators/datasource/templates/datasource.ts.ejs

```diff
@@ -5,7 +5,7 @@ import {
   ValueOrPromise,
 } from '@loopback/core';
 import {juggler} from '@loopback/repository';
-import * as config from './<%= jsonFileName %>';
+import config from './<%= jsonFileName %>';
```

Comment:
What will happen when we run `lb4 datasource` using the updated `lb4` CLI in and application that was created from an older CLI version and therefore has `esModuleInterop` disabled?

Reply:
The `@loopback/build` module will be released as a new major. Running `lb4 datasource` against an old project prompts users about the compatibility issue.